packages/snowpack/assets/hmr.js

```diff
@@ -146,6 +148,7 @@ socket.addEventListener('message', ({data: _data}) => {
   }
   debug('message: update', data);
   debug(data.url, Object.keys(REGISTERED_MODULES));
+  console.log(`${ESM_PREFIX} Updated`);
```

Comment:
Lets avoid logging these at both levels. Could changing `console.debug` to `console.log` address this issue instead? I think we've gone back and forth on this in the past, so let's commit to whichever we choose here and not change again.

Reply:
Personally, I only need to know that the update happened. All the rest of the information I feel like it more debugging stuff. That's why I was thinking we could split it up a bit and have it easily debuggable by enabling the debug/verbose logs.

If we want to go for a single place, I would say `console.log` and only do 3 basic outputs, `Updated`, `Reloaded`, and `Unknown Message`?

As @drwpow mentioned, this is slightly related to https://github.com/pikapkg/snowpack/pull/454 where the user wanted less verbose.

Reply:
#454 is pretty old, we used to log a ton of stuff (3 lines?) before we got it down to a single line. Not sure if that's still relevant here.

`message: update'` & `Updated` feel essentially identical, and then the `data` object is collapsed by default (in chrome at least) so that it's no extra noise unless you click into it. I'd prefer to keep this as-is and just send it back to console.log, now that we're no longer logging 3 messages at once.